src/app/modules/item/services/item-task-answer.service.ts

```diff
@@ -51,7 +51,7 @@ export class ItemTaskAnswerService implements OnDestroy {
         return readOnly ? of(formerAnswer) : this.loadFormerAsNewCurrentAnswer(route.id, attemptId, formerAnswer);
       }
 
-      return this.getCurrentAnswer(route.id, attemptId);
+      return readOnly ? of(formerAnswer) : this.getCurrentAnswer(route.id, attemptId);
```

Comment:
When `readOnly`, `formerAnswer` will be `null` but if someday it's not the service can already handle it this way.